blog.css

```diff
@@ -0,0 +1,313 @@
+body {
+    margin:0;
+    padding:0
+    
+}
+
+h1 {
+    font-size: 100px;
+}
+
+
+.blogtitle {
+    display: flex;
+    font-size: 105px;
+    font-weight: 700;
+    justify-content: center;
+    font-family: 'Peddana', serif;
+    color:darkorange;
+    margin-top: 50px;
+    text-decoration: none;
+   
+
+}
+
+.undertitle {
+    display: flex;
+    font-size: 18px;
+    font-weight: 400;
+    justify-content: center;
+    font-family: 'Peddana', serif;
+    color: black;
+    margin-top: 50px;
+}
+
+.linebreak {
+    width: 66vw;
+    size: 1px;
+    margin-top: 200px;
+    
+}
+.line2{
+    width: 66vw;
+    size: 1px;
+
+}
+
+.blogtext {
+    display: flex;
+    font-size: 28px;
+    justify-content: center;
+    font-weight: 600;
+    font-family: 'Peddana', serif;
+    color: black;
+    margin-top: 250px;
+}
+
+.blogimage{
+    height: 250px;
+    width: 700px;
+    display: block;
+    justify-content: center;
+    margin-left:380px ;
+    position: absolute;
+    top: 905px;
+}
+
+.h7{
+    display: flex;
+    position: relative;
+    font-size: 25px;
+    justify-content: center;
+    flex-direction: column;
+    font-weight: 900;
+    font-family: 'Peddana', serif;
+    color: black;
+    margin-top: 0px;
+    width: 65vw;
+    margin-left:30px;
+
+}
+
+.h8{
+    display: flex;
+    position: relative;
+    font-size: 25px;
+    justify-content: center;
+    flex-direction: column;
+    font-weight: 900;
+    font-family: 'Peddana', serif;
+    color: black;
+    margin-top: 0px;
+    width: 65vw;
+    margin-left:260px;
+    bottom:-40px;
+}
+
+.content {
+    display: flex;
+    font-size: 20px;
+    justify-content: center;
+    flex-direction: column;
+    font-weight: 250;
+    font-family: 'Peddana', serif;
+    color: black;
+    margin-top: 450px;
+    width: 65vw;
+    margin-left:215px;
+}
+
+.content2 {
+    display: flex;
+    font-size: 20px;
+    justify-content: center;
+    flex-direction: column;
+    font-weight: 250;
+    font-family: 'Peddana', serif;
+    color: black;
+    margin-top: 50px;
+    width: 65vw;
+    margin-left:215px;
+}
+    
+
+.footer {
+    display:flex;
+    position: absolute;
+    height: 500px;
+    width: 100vw;
+    background-color: lightgray;
+    margin-top:200px;
+    
+}
```

Comment:
you use a lot of `position: absolute` and `position: relative` through your footer (as well as a lot of other sections) that may be making your end product a little more off than expected.